server/routes/auth.py

```diff
@@ -20,6 +20,7 @@
     'old_pass': fields.String(required=True, description='Old password'),
     'new_pass': fields.String(required=True, description='New password')
 })
+users = {}
```

Comment:
Is this needed ? We don't see to be using it

Reply:
the users map is for returning a dictionary of all users connected